app.py

```diff
@@ -163,5 +165,37 @@ def setGridBoundries():
                               mimetype='application/json')
 
 
+@app.route('/export-data', methods=['POST'])
+def exportData():
+    if flask.request.is_json:
+        data = flask.request.get_json()
+
+        #TODO add/subtract/update information according to need
+        body = [{"studyName": data["flight_details"]["display_name"],
+                 "additionalInfo": {"features": data["features"],
+                                    "flight_id": data["flight_details"]["flight_id"],
+                                    "mission_start_time": data["flight_details"]["mission_start_time"]},
+                 "commonCropName": data["field_features"]["crop_type"],
+                 "contacts": [{"name": data["field_features"]["lead_scientist"],}],
+                 }]
+        
+        curl_request = ( f"curl --include \\\n"
+            f"     --request POST \\\n"
+            f"     --header \"Content-Type: application/json\" \\\n"
+            f"     --data-binary '{json.dumps(body, indent = 2)}' \\\n"
```

Comment:
can you check if indented json works? may cause errors in bash

Reply:
Indented json works in linux. In windows, the command for new line is different. Instead of '\' it is '^'. So even though the indent might work, the rest of the command probably won't work.